### DISCUSSION TITLE
Sending special characters to the VT

### DISCUSSION THREAD

**Discussion by thiagozf1** [Open-Agriculture/AgIsoStack-plus-plus#299] [Q&A]
Just a quick question right here, how can I send special characters (UTF-8) with `send_change_string_value` ?

**Comment by ad3154** [comment]
I believe `send_change_string_value` would work just as it would with ascii. The only difference really is that the length would be the number of bytes in the string instead of the number of characters, and you'd have to prefix a string literal with "u8".

```
yourVirtualTerminalClient->send_change_string_value(123, u8"☕");
```
This compiles for example and should work as long as the terminal supports the specific character, which it may not.

**Reply by ad3154** [reply]
Though, I will also say it is better practice to change the output string to use a different string variable or some other approach that isn't `send_change_string_value`, otherwise having multiple translations/language pools becomes more difficult.
